src/ts/components/label.ts

```diff
@@ -40,8 +45,11 @@ export class Label<Config extends LabelConfig> extends Component<Config> {
   }
 
   protected toDomElement(): DOM {
-    let labelElement = new DOM('span', {
+    const tagName = this.config.for != null ? 'label' : 'span';
```

Comment:
To handle also `undefined` values
```suggestion
    const tagName = Boolean(this.config.for) ? 'label' : 'span';
```

Reply:
`!= null` is falsy for `null` and `undefined` values, `!== null` is falsy only for `null`

Reply:
True. Missed the missing second `=` 👍